browser/ui/views/tabs/brave_tab_context_menu_contents.cc

```diff
@@ -47,7 +48,9 @@ BraveTabContextMenuContents::BraveTabContextMenuContents(
       TabRestoreServiceFactory::GetForProfile(browser_->profile());
   menu_runner_ = std::make_unique<views::MenuRunner>(
       model_.get(),
-      views::MenuRunner::HAS_MNEMONICS | views::MenuRunner::CONTEXT_MENU);
+      views::MenuRunner::HAS_MNEMONICS | views::MenuRunner::CONTEXT_MENU,
+      base::BindRepeating(&BraveTabContextMenuContents::OnMenuClosed,
```

Comment:
How about resetting `context_menu_contents_` in BraveBrowserTabStripController` with this callback?

Reply:
Good suggestion! I'll try to refactor in that way after PTO.